docs/forms/FORM_ERRORS.md

```diff
@@ -45,9 +26,8 @@ export interface FormError {
 }
 ```
 
-- __ElementError__
-This object contains all the errors associated to the html element.
-The identifier __htmlElementId__ can be used as an anchor link to focus on the html element on which the validation has failed
+- The `ElementError` object contains all the errors associated to the HTML element.
+The identifier `htmlElementId` can be used as an anchor link to focus on the HTML element where the validation failed.
```

Comment:
I guess this is a property from the ElementError interface but this could be stated directly